test-fixtures/slconfig.toml

```diff
@@ -0,0 +1,22 @@
+[server]
+  debug_trace = false
+
+[bluemix]
+  iam_url = "iam_rul"
+  iam_client_id = "bx"
+  iam_client_secret = "bx"
+  iam_api_key = "XXX"
+  refresh_token = ""
+  containers_api_route = "api_endpoint"
```

Comment:
container api endpoints are not required for vpc file, even refresh token also not required, please make sure we will use private iam endpoint as default else we will use whatever is present in the configuration